aiida_pseudo/groups/family/pseudo_dojo.py

```diff
@@ -0,0 +1,489 @@
+        },
+        'PseudoDojo/1.0/pbe/sr/standard/jthxml': {
+            'pseudo': 'http://www.pseudo-dojo.org/pseudos/paw_pbe_standard_xml.tgz',
+            'djrepo': 'http://www.pseudo-dojo.org/pseudos/paw_pbe_standard_djrepo.tgz'
+        },
+        'PseudoDojo/1.0/pbe/sr/stringent/jthxml': {
+            'pseudo': 'http://www.pseudo-dojo.org/pseudos/paw_pbe_stringent_xml.tgz',
+            'djrepo': 'http://www.pseudo-dojo.org/pseudos/paw_pbe_stringent_djrepo.tgz'
+        },
+        'PseudoDojo/1.0/lda/sr/standard/jthxml': {
+            'pseudo': 'http://www.pseudo-dojo.org/pseudos/paw_pw_standard_xml.tgz',
+            'djrepo': 'http://www.pseudo-dojo.org/pseudos/paw_pw_standard_djrepo.tgz'
+        },
+        'PseudoDojo/1.0/lda/sr/stringent/jthxml': {
+            'pseudo': 'http://www.pseudo-dojo.org/pseudos/paw_pw_stringent_xml.tgz',
+            'djrepo': 'http://www.pseudo-dojo.org/pseudos/paw_pw_stringent_djrepo.tgz'
+        }
+    }
+    # yapf: enable
+
+    @classmethod
+    def get_valid_labels(cls) -> Sequence[str]:
+        """Return the tuple of labels of all valid PseudoDojo configurations."""
+        configurations = set(cls.valid_configurations)
+        return tuple(cls.format_configuration_label(configuration) for configuration in configurations)
+
+    @classmethod
+    def format_configuration_label(cls, configuration: PseudoDojoConfiguration) -> str:
+        """Format a label for an `PseudoDojoFamily` with the required syntax.
+
+        :param configuration: the PseudoDojo configuration
+        :return: label
+        """
+        return cls.label_template.format(
+            version=configuration.version,
+            functional=configuration.functional,
+            relativistic=configuration.relativistic,
+            protocol=configuration.protocol,
+            pseudo_format=configuration.pseudo_format,
+        )
+
+    @classmethod
+    def get_urls(cls, label):
+        """Get the pseudo archive and djrepo archive urls for a given configuration label.
+
+        :param label: PseudoDojoConfiguration label
+        :return: dictionary with keys 'pseudo' for pseudo archive and 'djrepo' for djrepo archive urls
+        """
+        urls = cls.urls.get(label)
+        if urls is None:
+            raise ValueError(f'No urls were found for PseudoDojo configuration {label}')
+        return urls
+
+    @classmethod
+    def get_md5_from_djrepo(cls, djrepo, pseudo_type):
+        """Get the appropriate md5 hash from a DJREPO file.
+
+        :param djrepo: dictionary loaded from DJREPO JSON file
+        :reutnrs: md5 string
+        """
+        md5_key_mapping = {UpfData: 'md5_upf', Psp8Data: 'md5', PsmlData: 'md5_psml', JthXmlData: 'md5'}
+
+        md5_key = md5_key_mapping.get(pseudo_type)
+        if md5_key is None:
+            raise ValueError(f'pseudo type `{pseudo_type}` is unsupported by PseudoDojo djrepos.')
+        md5 = djrepo.get(md5_key)
+        if md5 is None:
```

Comment:
same here as for the urls. These should be `get(key, None)` or even better just do try/except the KeyError